src/osd/PG.cc

```diff
@@ -2539,7 +2539,7 @@ std::pair<ghobject_t, bool> PG::do_delete_work(
       epoch_t e = get_osdmap()->get_epoch();
       PGRef pgref(this);
       auto delete_requeue_callback = new LambdaContext([this, pgref, e](int r) {
-        dout(20) << __func__ << " wake up at "
+        dout(20) << "do_delete_work() [cb] wake up at "
```

Comment:
This change isn't relevant and can go in a separate commit.

Reply:
Fixed already